Functions-request-config.js

```diff
@@ -48,8 +48,4 @@ const requestConfig = {
   expectedReturnType: ReturnType.uint256,
 }
 
-if (requestConfig.secrets && !requestConfig.walletPrivateKey) {
```

Comment:
Why is this removed? Shouldn't we always require private key to be set?

Reply:
This check is already handled within `hardhat-config.js`